casaos-fix-docker-api-version/test-script.sh

```
@@ -0,0 +1,1786 @@
+
+echo "Calling get_docker_api_version..."
+get_docker_api_version
+TESTEOF
+  
+  chmod +x "$test_script"
+  
+  # Run with mocked docker
+  export PATH="$mock_dir:$PATH"
+  start_time=$(date +%s)
+  local output=$($test_script 2>&1)
+  local func_exit=$?
```

Shellcheck (0.11.0)

[warning] func_exit appears unused. Verify use (or export if used externally).

(SC2034)